src/webapi/html_elements/textarea.rs

```diff
@@ -0,0 +1,42 @@
+use webcore::value::{Value, Reference};
+use webapi::event_target::{IEventTarget, EventTarget};
+use webapi::node::{INode, Node};
+use webapi::element::{IElement, Element};
+use webapi::html_element::{IHtmlElement, HtmlElement};
+
+/// The HTML <textarea> element represents a multi-line plain-text editing control.
+///
+/// [(JavaScript docs)](https://developer.mozilla.org/en/docs/Web/HTML/Element/textarea)
+pub struct TextAreaElement( Reference );
+
+impl IEventTarget for TextAreaElement {}
+impl INode for TextAreaElement {}
+impl IElement for TextAreaElement {}
+impl IHtmlElement for TextAreaElement {}
+
+reference_boilerplate! {
+    TextAreaElement,
+    instanceof HTMLTextAreaElement
+    convertible to EventTarget
+    convertible to Node
+    convertible to Element
+    convertible to HtmlElement
+}
+
+impl TextAreaElement {
+    /// The value of the control.
+    #[inline]
+    pub fn value( &self ) -> Value {
```

Comment:
According to the [specs](https://html.spec.whatwg.org/multipage/form-elements.html#the-textarea-element) this should return a `String`.